src/ADAL.PCL.iOS/BrokerKeyHelper.cs

```diff
@@ -73,7 +73,12 @@ internal static byte[] GetRawBrokerKey()
                     ValueData = byteData
                 };
 
-                SecStatusCode code = SecKeyChain.Add(record);
+                var result = SecKeyChain.Add(record);
```

Comment:
This isn't used in the Mac backend. It was causing a warn-as-error build failure in the iOS backend.